packages/adapter/addon/-private/fastboot-interface.ts

```diff
@@ -0,0 +1,24 @@
+export interface Request {
+  method: string;
+  body: unknown;
+  cookies: unknown;
```

Comment:
Lets have a bit more specific types here for objects

Reply:
What do you have in mind here?  

Re: cookies...

https://github.com/ember-fastboot/fastboot/blob/9b69cf5a6f093d8068c4108684471dcac296e375/src/fastboot-request.js#L42

An interface for headers I guess could include a few like Content-Type and the like with a catch all.  Is this the idea?

Reply:
Just to confirm, you generally want only the used properties declared as a part of this interface?